### PR TITLE
N14: Drawing lines

### PR DESCRIPTION
**Pull request by pum-purum-pum-pum**

### CODE REVIEW

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -62,6 +62,23 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+A small note about drawing lines and chains with signed distance fields.
+The resulted lines are nice looking on edges and have rounded corners.
+Chains for this article assumed to be opaque.
+
+[Online demo](https://pum-purum-pum-pum.github.io/lines/)
+
+[Source code](https://github.com/pum-purum-pum-pum/Lines)
+
+Discussions:
+[twitter](https://twitter.com/VladZhukov0/status/1309843665594925058)
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:78]:
```suggestion
_Discussions: [twitter](https://twitter.com/VladZhukov0/status/1309843665594925058)_
```

File: content/posts/newsletter-014/index.md [hunk 2]
```diff
@@ -62,6 +62,23 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Is it easy to draw a line?][Lines]
+
+![image/GIF description](lines_hexstrat.png)
+_Example of result chains_
+A small note about drawing lines and chains with signed distance fields.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:69]:
```suggestion

A small note about drawing lines and chains with signed distance fields.
```

**Review by 17cupsofcoffee — APPROVED** [review]
Content looks good, just needs a couple of formatting tweaks.

**Review by ozkriff — APPROVED** [review]
:+1: (fixed the image & made a few style tweaks for consistency)

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**